src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let newState = { ...state };
+
+  actions.forEach((action) => {
```

Comment:
According to the checklist:
_[CODE STYLE]: Use switch statement if you have limited amount of conditions._

Try refactoring using **switch**